--- a/src/Makefile	2020-03-04 06:44:05.000000000 +0400
+++ b/src/Makefile	2025-04-11 22:32:12.798581265 +0400
@@ -9,7 +9,7 @@
 OBJS = main.o mainwindow.o inifile.o png.o memory.o canvas.o otherwindow.o mygtk.o\
 	viewer.o polygon.o layer.o info.o wu.o prefs.o ani.o mtlib.o\
 	toolbar.o channels.o csel.o shifter.o spawn.o font.o fpick.o icons.o\
-	cpick.o thread.o vcode.o
+	fpick.o icons.o cpick.o thread.o vcode.o
 
 $(BIN): $(OBJS)
 	$(CC) $(OBJS) -o $(BIN) $(LDFLAGS)
