--- src/SConstruct.orig	2025-03-10 20:24:34.268767289 +0400
+++ src/SConstruct	2025-03-10 20:25:30.906716062 +0400
@@ -1,4 +1,4 @@
-
+import os
 env = Environment(tools = ['default', 'textfile'])
 
 name = 'helianthus'
@@ -12,7 +12,12 @@
 WITH_FULL_UNICODE_FONT = ARGUMENTS.get('WITH_FULL_UNICODE_FONT', 0)
 
 opts = Variables(name + '.conf')
-opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr/local', PathVariable.PathAccept))
+destdir = os.environ.get('DESTDIR', '')  # Ensure DESTDIR is respected
+opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr', PathVariable.PathAccept))
+
+if destdir:
+    env['PREFIX'] = destdir + env['PREFIX']
+
 opts.Update(env)
 opts.Save(name + '.conf', env)
 
