diff -ur a/src/SConstruct b/src/SConstruct
--- a/src/SConstruct	2025-03-10 20:59:46.400114320 +0400
+++ b/src/SConstruct	2025-03-10 20:47:43.367319536 +0400
@@ -12,7 +12,12 @@
 WITH_FULL_UNICODE_FONT = ARGUMENTS.get('WITH_FULL_UNICODE_FONT', 0)
 
 opts = Variables(name + '.conf')
-opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr/local', PathVariable.PathAccept))
+destdir = os.environ.get('DESTDIR', '')  # Ensure DESTDIR is respected
+opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr', PathVariable.PathAccept))
+
+if destdir:
+    env['PREFIX'] = destdir + env['PREFIX']
+
 opts.Update(env)
 opts.Save(name + '.conf', env)