diff -ur a/src/SConstruct b/src/SConstruct
--- a/src/SConstruct	2025-03-10 20:59:46.400114320 +0400
+++ b/src/SConstruct	2025-03-10 20:47:43.367319536 +0400
@@ -1,5 +1,5 @@
-
+import os
 env = Environment(tools = ['default', 'textfile'])
 
 name = 'helianthus'
 description = 'Helianthus is a library to create small games'
 version = '0.1.0'
@@ -11,7 +11,12 @@
 DEBUG = ARGUMENTS.get('DEBUG', 0)
 WITH_FULL_UNICODE_FONT = ARGUMENTS.get('WITH_FULL_UNICODE_FONT', 0)
 
 opts = Variables(name + '.conf')
-opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr/local', PathVariable.PathAccept))
+destdir = os.environ.get('DESTDIR', '')  # Ensure DESTDIR is respected
+opts.Add(PathVariable('PREFIX', 'Directory to install under', '/usr', PathVariable.PathAccept))
+
+if destdir:
+    env['PREFIX'] = destdir + env['PREFIX']
+
 opts.Update(env)
 opts.Save(name + '.conf', env)